src/com/lftechnology/java/training/sanish/numberprinter/NumberPrinter.java

```diff
@@ -0,0 +1,32 @@
+package com.lftechnology.java.training.sanish.numberprinter;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Print number for 1-10
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class NumberPrinter {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)